### PR TITLE
Changes width, height of Licence

### PR DESCRIPTION
**Pull request by jahnvigupta**
**Fixes**

Fixes #[8] by @[dhruvkb]

**Description**

The PR makes the width and height of cc_zero.svg and publicdomain.svg same as other lincenses.

**Type of PR**

This PR is a feature.

**Technicalities**

**Tests**

**Screenshots**

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
@jahnvigupta merely changing the height and width of the licenses does not scale them as they're SVGs. SVGs have a different interpretation of `height`, `width` and `viewBox` attributes. You will need [Inkscape](https://inkscape.org/) or [Illustrator](https://www.adobe.com/products/illustrator.html) to actually resize the SVG.

File: src/assets/license_badges/big/cc_zero.svg [hunk 1]
```diff
@@ -1,98 +1,154 @@
...
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="Layer_1"
+   x="0px"
+   y="0px"
+   width="120"
+   height="42"
+   viewBox="-0.5 -0.101 120 42"
```
Comment (dhruvkb) [inline src/assets/license_badges/big/cc_zero.svg]: For ease of use, we recommend all SVGs to have `viewBox="0 0 120 42"`.

File: src/assets/license_badges/big/publicdomain.svg [hunk 2]
```diff
@@ -1,86 +1,140 @@
...
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="Layer_1"
+   x="0px"
+   y="0px"
+   width="120"
+   height="42"
+   viewBox="0.995 1.495 120 42"
```
Comment (dhruvkb) [inline src/assets/license_badges/big/publicdomain.svg]: For ease of use, we recommend all SVGs to have viewBox="0 0 120 42".

**Review by dhruvkb — CHANGES_REQUESTED** [review]
By ensuring consistent `viewBox` values, we can eliminate accidental cropping.

**Comment by jahnvigupta** [comment]
@dhruvkb Please have a look.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**